TAScheduler/migrations/0001_initial.py

```diff
@@ -0,0 +1,97 @@
+# Generated by Django 4.2.7 on 2023-11-19 03:39
+
+import django.core.validators
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Course',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('course_id', models.IntegerField()),
+                ('semester', models.CharField(max_length=11)),
+            ],
```

Comment:
I will submit the "updated diagram" so it makes sense that it's not present here ... but course should have "name" (String) "description"(string) "Number of labs"(int)  "semester"(string)  "times"(Datetime?) "modality"(String

Reply:
Times should not be in course. They are apart of sections and courses never meet sections do